CMakeLists.txt

```diff
@@ -125,6 +125,8 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "^(GNU|Clang)$")
   np_detect_options(ASMJIT_CC_OPTIONS
     "-fno-exceptions"
     "-fno-tree-vectorize"
+    "-Wno-inconsistent-missing-override"
+    "-Wno-undefined-inline"
```

Comment:
* https://github.com/Barro/compiler-warnings
* https://github.com/Tuplanolla/gcc-weverything

Clang has a handy option called `-Weverything`, which enables every warning built into the compiler.